src/lib/y2network/interface_config_builder.rb

```diff
@@ -232,6 +240,23 @@ def aliases
     # @param value [Array<Hash>] see #aliases for hash values
     def aliases=(value)
       @aliases = value
+
+      # connection config
+      # keep only default as aliases does not handle default ip config
+      @connection_config.ip_configs.delete_if { |c| c.id != "" }
```

Comment:
We could add `ConnectionConfig::Base#main_ip_config` and `ConnectionConfig::Base#aliases` (or something like this) so you do not have to worry about this.

Reply:
yep, when added, feel free to change this code ;)